app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -335,8 +335,8 @@ public function store(HttpRequest $request)
 
         try {
             $topic = Topic::createNew($forum, $params, $poll ?? null);
```

Comment:
`Topic::addPost` called through `Topic::createNew` doesn't explode when failing to save, so maybe should do something about it while changing all this?